openapi/multi-yaml/paths/payment_intents.yaml

```diff
@@ -10,7 +10,7 @@ post:
   parameters:
     - $ref: ../components/parameters/idempotency-key-header.yaml
     - $ref: ../components/parameters/authorization-header.yaml
-    - $ref: ../components/parameters/sub-account.yaml
+    - $ref: ../components/parameters/sub-account-required.yaml
```

Comment:
Why not make `sub-account.yaml` required instead of creating a new file? Are there cases a sub-account is still optional?

Reply:
Yeah, there are still cases where it's optional